### PR TITLE
[notification] webhook for alert

### PR DESCRIPTION
**Pull request by qnkhuat**
Feature branch
Epic: https://github.com/metabase/metabase/issues/43822

### CODE REVIEW

**Comment by qnkhuat** [comment]
@npfitz could you help resolve the merge conflicts?

**Comment by qnkhuat** [comment]
Code already approved in sub-PRs, just need another sign off from product

**Review by npfitz — APPROVED** [review]

File: .clj-kondo/config.edn [hunk 1]
```diff
@@ -808,6 +812,7 @@
   potemkin/deftype+                                                                     clojure.core/deftype
   toucan.db/with-call-counting                                                          clojure.core/fn
   toucan2.core/with-call-count                                                          clojure.core/fn}
+ >>>>>>> master
```
Comment (npfitz) [inline .clj-kondo/config.edn:815]: @qnkhuat I think we missed this haha

**Comment by qnkhuat** [comment]
accidently made a bad merge conflict which includes all commits on master. fixed it

**Pull request merged**